.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Bake target'
+        required: true
+        default: 'mainline'
+      latest:
+        description: 'Create latest tag'
+        required: true
+        default: 'false'
+      dry-run:
```

Comment:
would be good to override qemu in here. For test releases, rollbacks etc.

Reply:
I had to use an HCL function to handle default values. Let me know if it looks good to you: https://github.com/tonistiigi/binfmt/pull/24/commits/7bf70fe0e75c5623d221aafacdfb23d0b4de87e1